README.md

```diff
@@ -150,4 +150,14 @@ Now that you have a basic sense of how to read these error messages, as well as
 
 When you're done, run `learn` to confirm all tests are passing.
 
+#### What's the Difference between `gets`, `gets.chomp`, and `gets.strip`?
+
+When getting user input, you may want to remove whitespace. The `gets.chomp` method removes whitespace AFTER the user input. The `gets.strip` method removes whitespace BEFORE and AFTER the user input. The `gets` method by itself does not remove any whitespace or newlines from the user input. Your tests will pass using any of these methods, however, be aware that they perform different actions and are not always interchangeable for every use-case.
```

Comment:
Lastly, please try to wrap 80col when doing edits. I can target sentences w/ feedback.